src/goMain.ts

```diff
@@ -367,8 +367,29 @@ export function activate(ctx: vscode.ExtensionContext): void {
 	}));
 
 	ctx.subscriptions.push(vscode.commands.registerCommand('go.show.commands', () => {
-		vscode.window.showQuickPick(getExtensionCommands().map(x => x.title)).then(cmd => {
-			let selectedCmd = getExtensionCommands().find(x => x.title === cmd);
+		let extCommands = getExtensionCommands()
+		extCommands.push({
+			command : 'editor.action.goToTypeDefinition',
+			title : 'Go : Go to Type Definition',
+			description : 'Go : invokes the GoDefinitionProvider'
```

Comment:
- This should be `editor.action.goToDeclaration` with title `Go to Definition`.
- We can skip the description. It doesnt get used in the `Show all commands` feature
- Let's not include the prefix `Go:` in the title. 
The prefix is for commands explicitly registered and implemented by the Go extension.
Where as these 4 commands that we are adding is registered in core VS Code an is only implemented by the Go extension

Reply:
1. will change to `editor.action.goToDeclaration` . 
2. description i kept for sake of documentation. it was hard for me to see the connection between the various `Providers` in the vscode-go extension and their corresponding internal commands. thats why initially i started with creating a new command
3. ok will remove the 'Go:' prefix, else it may be thought of a separate command.
in fact i was looking for some api to get the complete command object for these four commands, so that title, description etc can be reused directly. i couldn't find so created the literals.